nnvm/tests/python/frontend/keras/test_forward.py

```diff
@@ -171,13 +222,18 @@ def test_forward_mobilenet():
     test_forward_softmax()
     test_forward_softrelu()
     test_forward_leaky_relu()
+    test_forward_prelu()
+    test_forward_elu()
+    test_forward_selu()
+    test_forward_thresholdedrelu()
+    test_forward_softsign()
+    test_forward_hardsigmoid()
```

Comment:
Can you merge these many activation layer tests into one test_forward_activation()?

Reply:
Better to merge these activation layer tests this way to reduce code duplication:

```python

def test_forward_activation():
    data = keras.layers.Input(shape=(32,32,3))
    act_funcs = [keras.layers.Activation('softmax'),
                 keras.layers.Activation('softplus'),
                 keras.layers.LeakyReLU(alpha=0.3),
                 keras.layers.Activation(keras.applications.mobilenet.relu6)]
    for act_func in act_funcs:
        x = act_func(data)
        x = keras.layers.GlobalMaxPooling2D()(x)
        keras_model = keras.models.Model(data, x)
        verify_keras_frontend(keras_model)
```